.env

```diff
@@ -1,3 +1,13 @@
+### IMPORTANT
+
+### - This `.env` file is tracked by git.
+
+### - Only add default variables to this file
+
+### - Create an `.env.local` to add your own variables.
+
 NEXT_PUBLIC_MGNL_HOST=http://localhost:8080/magnoliaAuthor
 NEXT_PUBLIC_MGNL_LANGUAGES="en de"
-NEXT_PUBLIC_MGNL_NODE_NAME="/nextjs-ssg-minimal"
\ No newline at end of file
+NEXT_PUBLIC_MGNL_NODE_NAME="/nextjs-ssg-minimal"
+PLASMIC_PROJECT_ID="9fuL7F7jJuiHG9rz5yy9QV"
```

Comment:
This may end up needing to be prefixed with NEXT_PUBLIC_ to be available for static generation.
Since we haven't had a public Magnolia instance the static generation has not been tested.